EasyCommands/CommandParsers/ParameterProcessors.cs

```diff
@@ -211,10 +213,11 @@ public static class ParameterProcessorRegistry {
                 //AmgiguousSelectorPropertyProcessor
                 new BranchingProcessor<SelectorCommandParameter>(
                     TwoValueRule<SelectorCommandParameter,PropertyCommandParameter,DirectionCommandParameter>(
-                        optionalEither<PropertyCommandParameter>(), optionalEither<DirectionCommandParameter>(),
-                        (s,p,d) => new VariableCommandParameter(new AggregatePropertyVariable(PropertyAggregatorType.VALUE, s.Value, p.HasValue() ? p.GetValue().Value : (PropertyType?)null, d.HasValue() ? d.GetValue().Value : (DirectionType?)null))),
+                        requiredEither<PropertyCommandParameter>(), optionalEither<DirectionCommandParameter>(),
+                        (s,p,d) => new VariableCommandParameter(new AggregatePropertyVariable(PropertyAggregatorType.VALUE, s.Value, p.GetValue().Value, d.HasValue() ? d.GetValue().Value : (DirectionType?)null))),
                     TwoValueRule<SelectorCommandParameter,PropertyCommandParameter,DirectionCommandParameter>(
                         optionalEither<PropertyCommandParameter>(), optionalEither<DirectionCommandParameter>(),
+                        (s,p,d) => p.HasValue() || d.HasValue(),//Must have at least one!
```

Comment:
This was a small bugfix.  BlockCommand cannot parse (fails with exception) if you try to give it a selector without property OR direction, yet the rules allowed it.